pkg/apis/ipfix/ies.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2025 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipfix
+
+import (
+	"iter"
+
+	"github.com/vmware/go-ipfix/pkg/registry"
+)
+
+type IPFamilyType string
+
+const (
+	IPFamilyBoth IPFamilyType = ""
+	IPFamilyIPv4 IPFamilyType = "IPv4"
+	IPFamilyIPv6 IPFamilyType = "IPv6"
+)
+
+type InfoElement struct {
+	Name           string
+	EnterpriseName string
+	EnterpriseID   uint32
+	IPFamily       IPFamilyType
+}
+
+// AllInfoElements is the ordered list of all Information Elements exported by the FlowExporter.
+// Only add new elements to the end of this list!
```

Comment:
So we should append any new element at the end of the whole list? No matter which category (IANA vs Antrea) it is?